src/environments/environment.ts

```diff
@@ -12,6 +12,11 @@ export const environment = {
   mapboxAccessToken:
     'pk.eyJ1IjoiZmVsaXhhZXRlbSIsImEiOiI2MmE4YmQ4YjIzOTI2YjY3ZWFmNzUwOTU5NzliOTAxOCJ9.nshlehFGmK_6YmZarM2SHA',
   photoQuality: 10,
+  // VR world
+  initialAvatarLoc: {
+    lng:224 / 111000,
+    lat:74 / 112000
+  }
```

Comment:
Please add the `initialAvatarLoc` object to the `src/environments/environment.prod.ts` file as well

Reply:
I've updated it.